src/controllers/dlgprefcontroller.cpp

```diff
@@ -749,3 +750,11 @@ void DlgPrefController::openScript() {
         }
     }
 }
+
+
+bool DlgPrefController::presetInfoLessThan(const PresetInfo &a, const PresetInfo &b) {
```

Comment:
nitpick: Please add this to the anonymous namespace at the top of the file and give it a more descriptive name e.g. "presetInfoNameComparator" or similar. No need for it to be a static class member. thanks!